packages/expo-barcode-scanner/android/src/main/java/expo/modules/barcodescanner/BarCodeScannerView.java

```diff
@@ -60,11 +65,55 @@ public void onViewAdded(View child) {
 
   public void onBarCodeScanned(BarCodeScannerResult barCode) {
     EventEmitter emitter = mModuleRegistry.getModule(EventEmitter.class);
-    BarCodeScannedEvent event = BarCodeScannedEvent.obtain(this.getId(), barCode);
+    convertResult(barCode);
+    BarCodeScannedEvent event = BarCodeScannedEvent.obtain(this.getId(), barCode, getReactNativePointDensity());
     emitter.emit(this.getId(), event);
   }
 
+  // returns number of pixels in one React Native Point
```

Comment:
```suggestion
```

Density helps us convert between pixels and DP (Display-Indepent Points), which is a normal Android unit of measure https://developer.android.com/guide/topics/resources/more-resources#Dimension